cache_images/fedora_packaging.sh

```diff
@@ -194,6 +194,12 @@ DOWNLOAD_PACKAGES=(\
 msg "Installing general build/test dependencies"
 bigto $SUDO dnf install -y "${INSTALL_PACKAGES[@]}"
 
+timebomb 20240214 "Gaaaaaaaaah! Kludge to emergency-get netavark 1.10.3"
+arch=$(uname -m)
+fedoraversion=$(awk -F= '$1 == "VERSION_ID" { print $2 }' </etc/os-release)
+bigto $SUDO dnf install -y https://kojipkgs.fedoraproject.org//packages/netavark/1.10.3/1.fc${fedoraversion}/${arch}/netavark-1.10.3-1.fc${fedoraversion}.${arch}.rpm
+timebomb 20240214 "^^^^^^^^ delete everything above here"
```

Comment:
There's a bodhi for fc40 but none for f38/f39. This seemed important.

Reply:
I definitely won't remember that, would you consider adding a comment (here or in a followup PR)?

Reply:
netavark-1.10.3 is now in testing, and will presumably soon be in stable. Since this code is timebombed, I believe it is safe to merge as-is.